cluster/manifests/audittrail-adapter/credentials.yaml

```diff
@@ -12,3 +12,20 @@ spec:
      audittrail:
        privileges: []
 {{- end }}
+{{- if .Cluster.ConfigItems.audittrail_nakadi_url }}
+---
```

Comment:
We should also add an entry in `cluster/manifest/deletions.yaml` Such that this PCS is deleted if we set `audittrail_nakadi_url=""` after having set an URL.

Reply:
Added. I went with the `if eq ... ""` syntax as opposed to `if not ...` as it looked more in-line with the rest of the `deletions.yaml` file content.